tools/hangar/__snapshots__/test_corpus/sdk_tests/container/container.test.w_test_sim.md

```diff
@@ -2,9 +2,17 @@
 
 ## stdout.log
 ```log
+[INFO] get echo | Pulling hashicorp/http-echo...
+[INFO] get echo | Container hashicorp/http-echo started
+[INFO] get app | Building my-app:a9ae83b54b1ec21faa1a3255f05c095c from ./my-docker-image...
+[INFO] get app | Container my-app:a9ae83b54b1ec21faa1a3255f05c095c started
+[INFO] get echo | Container my-app:a9ae83b54b1ec21faa1a3255f05c095c started
+[INFO] get app | Container hashicorp/http-echo started
 [INFO] get echo | bang
 [INFO] get echo | 
 [INFO] get app | Hello, Wingnuts!
+[WARNING] get echo | Timeout waiting for container root/env0/my-app to shutdown, removing forcefully
+[WARNING] get app | Timeout waiting for container root/env1/my-app to shutdown, removing forcefully
```

Comment:
Should this be happening every time?

Reply:
Yes because this container  doesn't react properly to signals. But that's okay I like that we are testing this.